pangeo/values.yaml

```diff
@@ -56,25 +56,52 @@ jupyterhub:
 dask-gateway:
   # dask-gateway configuration goes here
   # See https://github.com/dask/dask-gateway/blob/master/resources/helm/dask-gateway/values.yaml
+  # The values here are optimized for deployment within a jupyterhub.
   gateway:
-    clusterManager:
```

Comment:
This has been replaced with `gateway.image`. I think the default of `daskgateway/dask-gateway-server` is fine for us.